lib/my_john_deere_api/client.rb

```diff
@@ -45,7 +45,7 @@ def initialize(api_key, api_secret, options = {})
 
     def accessor
       return @accessor if defined?(@accessor)
-      @accessor = OAuth::AccessToken.new(consumer.user_get, access_token, access_secret)
+      @accessor = NetHttpRetry::Decorator.new(OAuth::AccessToken.new(consumer.user_get, access_token, access_secret))
```

Comment:
It's possible that I'm missing something, but it seems like you still need a mechanism to get configuration options from the outside application through the client and down into the retry decorator.

Reply:
You are correct, sir. I'll make the fix.